test/unit/uk/gov/hmrc/digitalservicestaxfrontend/TestInstances.scala

```diff
@@ -0,0 +1,434 @@
+    "KW23YOUF8981762754308793114869",
+    "KZ830907614112967961",
+    "LB52372840451692007088329912",
+    "LC55HEMM000100010012001200023015",
+    "LI1935676368061346475",
+    "LT192412904756279442",
+    "LU951073196477746242",
+    "LV88HAGO9615590045208",
+    "MC3212096543766737867569392",
+    "MD9881991481700015080266",
+    "ME36752723963983139414",
+    "MK31948483764450559",
+    "MR5442207381102036325668909",
+    "MT84AIWA00813109843252965695890",
+    "MU65OSUR4348845783494200142ABC",
+    "NL64VCYA2607250706",
+    "NO2451742753161",
+    "PK13LCJY3078553597017331",
+    "PL64778828144458067930515057",
+    "PS38LEKA813957891201287138525",
+    "PT87847144161279936872891",
+    "QA58DOHB00001234567890ABCDEFG",
+    "RO17KRBO3212835705756123",
+    "RS80246113647338221492",
+    "SA0979413508515371577583",
+    "SC18SSCB11010000000000001497USD",
+    "SE0651297191201320278580",
+    "SI75412441865827872",
+    "SK7336568401664473733427",
+    "SM67H0718392993037614346095",
+    "ST32000200010192194210112",
+    "SV62CENR00000000000000700025",
+    "TL380080012345678910157",
+    "TN6199977105796904072050",
+    "TR330006100519786457841326",
+    "TR888859050625760496700846",
+    "UA213223130000026007233566001",
+    "VA59001123000012345678",
+    "BE71096123456769",
+    "FR7630006000011234567890189",
+    "DE91100000000123456789",
+    "GR9608100010000001234567890",
+    "RO09BCYP0000001234567890",
+    "SA4420000001234567891234",
+    "ES7921000813610123456789",
+    "CH5604835012345678009",
+    "GB98MIDL07009312345678",
+    "VG14NDUM4605555206975725",
+    "XK051212012345678906",
+    "BE71 0961 2345 6769",
+    "FR76 3000 6000 0112 3456 7890 189",
+    "DE91 1000 0000 0123 4567 89",
+    "GR96 0810 0010 0000 0123 4567 890",
+    "RO09 BCYP 0000 0012 3456 7890",
+    "SA44 2000 0001 2345 6789 1234",
+    "ES79 2100 0813 6101 2345 6789",
+    "CH56 0483 5012 3456 7800 9",
+    "GB98 MIDL 0700 9312 3456 78"
+  )
+
+  implicit val arbIban: Arbitrary[IBAN] = Arbitrary {
+    Gen.oneOf(ibanList).map(IBAN.apply)
+  }
+
+  implicit val arbKickout: Arbitrary[Kickout] = Arbitrary {
+    nonEmptyString.map(Kickout.apply)
+  }
+  implicit val arbAccountName: Arbitrary[AccountName] = Arbitrary {
+    RegexpGen.from(AccountName.regex).map(AccountName.apply)
+  }
+
+  implicit val arbAccountNumber: Arbitrary[AccountNumber] = Arbitrary {
+    RegexpGen.from(AccountNumber.regex).map(AccountNumber.apply)
+  }
+
+  implicit val arbBuildingSocietyRollNumber: Arbitrary[BuildingSocietyRollNumber] = Arbitrary {
+    RegexpGen.from(BuildingSocietyRollNumber.regex).map(BuildingSocietyRollNumber.apply).retryUntil(x => x == x.trim)
+  }
+
+  implicit val arbSortCode: Arbitrary[SortCode] = Arbitrary {
+    RegexpGen.from(SortCode.regex).map(SortCode.apply)
+  }
+
+  implicit val argRestrictedString: Arbitrary[RestrictiveString] = Arbitrary(
+    RestrictiveString.gen
+    //    RegexpGen.from("""^[0-9a-zA-Z{À-˿’}\\- &`'^._|]{1,255}$""")
```

Comment:
Is this comment required?

Reply:
The comment that you mentioned I think we should keep them

Reply:
The whole issue with RFC compliant emails is, I think, estate-wide and I've never heard a satisfactory solution. 
We could validate against the RFC, but then we would be admitting email addresses that HoD's would almost certainly choke upon.

Reply:
Interesting, I'll take a look into this in the mean time. Sounds quite interesting.